src/Processors/Formats/LazyOutputFormat.h

```diff
@@ -19,9 +19,9 @@ class LazyOutputFormat : public IOutputFormat
 
     String getName() const override { return "LazyOutputFormat"; }
 
-    Block getBlock(UInt64 milliseconds = 0);
-    Block getTotals();
-    Block getExtremes();
+    Chunk getChunk(UInt64 milliseconds = 0);
```

Comment:
https://github.com/ClickHouse/ClickHouse/blob/3537c1a65e8a144454b1b9e7bac48238a023108e/src/Processors/Formats/LazyOutputFormat.h#L11

Need to correct the comment, but after all tests are executed.

Reply:
Sure